resources/usa_garage/server.lua

```diff
@@ -136,6 +136,41 @@ AddEventHandler("garage:vehicleSelected", function(vehicle, business, playerCoor
 		else
 			TriggerClientEvent("usa:notify", usource, "~y~STATE IMPOUND: ~s~Your vehicle is impounded and can be retrieved for ~y~$"..IMPOUND_FEE..".00~s~!")
 		end
+	elseif vehicle.adv_impounded == true then
+		if (vehicle.released_on - os.time() > 0) then
+			local time_until = round((vehicle.released_on - os.time())/86400, 0)
```

Comment:
```suggestion
			local time_until = math.floor((vehicle.released_on - os.time())/86400, 0)
```

built in and has no decimals